src/bin/trace-verifier/utils.rs

```diff
@@ -10,11 +13,11 @@ pub fn verify(
     fork_config: &HardforkConfig,
     disable_checks: bool,
     log_error: bool,
-) -> bool {
+) -> Result<bool, VerificationError> {
```

Comment:
now that we use a result, we can switch to:
```suggestion
) -> Result<(), VerificationError> {
```